src/CoreWCF.Http/tests/Services/ContractBehaviorSerivce.cs

```diff
@@ -0,0 +1,86 @@
+﻿using CoreWCF;
+using CoreWCF.Dispatcher;
+using ServiceContract;
+
+namespace Services
+{
+    public abstract class IContractBehaviorBasic_ServiceBase
+    {
+        public string ServiceContractMethod(string s)
+        {
+            //TODO: validate behavior
+            //BehaviorInvokedVerifier.ValidateServiceInvokedBehaviors(OperationContext.Current.Host.Description);            
```

Comment:
@mconnew This case is testing IContractBehavior, when validating behavior for service, the below object is unavailable: `OperationContext.Current.Host.Description`. There's `ServiceHostBase host `in `CoreWCF.OperationContext` and it is private, is there any plan to expose the property? If not do you have any suggestion on how to validate the behaviors after service be hosted?

Reply:
I do plan to expose the property. I've tried not to expose things unless they are tested. Could you make that property public as this test indirectly tests it.

Reply:
Updated: add public property OperationContext.Current.Host for test.